ros_ws/src/vision/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ find_package(catkin REQUIRED COMPONENTS
 ## catkin specific configuration ##
 ###################################
 catkin_package(
```

Comment:
This should not have been committed as it was fixed in #69.  Please check that you aren't adding unrelated files when adding to your commit.